stellar_sdk/muxed_account.py

```diff
@@ -0,0 +1,112 @@
+from .exceptions import ValueError, MuxedEd25519AccountInvalidError
+from .strkey import StrKey, encode_check, decode_check
+from .utils import check_ed25519_public_key
+from .xdr import Xdr
+
+__all__ = ["MuxedAccount"]
+
+
+class MuxedAccount:
```

Comment:
Hi @abuiles, do you think it is worth adding the `MuxedAccount ` class to js-stellar-base? I would love to hear your opinion.